website/docs/cloud-docs/workspaces/settings/deletion.mdx

```diff
@@ -25,15 +25,33 @@ Before queuing a destroy plan, enable the **Allow destroy plans** toggle setting
 @include 'tfc-package-callouts/ephemeral-workspaces.mdx'
 <!-- END: TFC:only name:pnp-callout -->
 
-Automated destroy runs are available as part of the ephemeral workspaces feature.
+You can schedule a run to destroy all resources in a workspace at a specific
+date and time. This ephemeral workspace option reduces your infrastructure
```

Comment:
> This ephemeral workspace option

just for context: Ephemeral Workspaces is an entitlement, which can technically have many features – but right now the only feature under this entitlement is the auto destroy functionality. swapping out `option` -> `entitlement` may be more clear here that it's tied to pricing/subscription. I don't feel strongly about this though, just sprinkling in context.